2.0/README.md

```diff
@@ -0,0 +1,500 @@
+```
+
+```
+id = CommandInteger & 0x7
+```
+
+```
+count = CommandInteger >> 3
+```
+
+A command ID specifies one of the following commands:
+
+|  Command     |  Id  | Parameters    | Parameter Count |
+| ------------ |:----:| ------------- | --------------- |
+| MoveTo       | `1`  | `dX`, `dY`    | 2               |
+| LineTo       | `2`  | `dX`, `dY`    | 2               |
+| ClosePath    | `7`  | No parameters | 0               |
+
+##### Example Command Integers
+
+| Command   |  ID  | Count | CommandInteger | Binary Representation `[Count][Id]`      |
+| --------- |:----:|:-----:|:--------------:|:----------------------------------------:|
+| MoveTo    | `1`  | `1`   | `9`            | `[00000000 00000000 0000000 00001][001]` |
+| MoveTo    | `1`  | `120` | `961`          | `[00000000 00000000 0000011 11000][001]` |
+| LineTo    | `2`  | `1`   | `10`           | `[00000000 00000000 0000000 00001][010]` |
+| LineTo    | `2`  | `3`   | `26`           | `[00000000 00000000 0000000 00011][010]` |
+| ClosePath | `7`  | `1`   | `15`           | `[00000000 00000000 0000000 00001][111]` |
+
+
+#### 4.3.2. Parameter Integers
+
+Commands requiring parameters are followed by a `ParameterInteger` for each parameter required by that command. The number of `ParameterIntegers` that follow a `CommandInteger` is equal to the parameter count of a command multiplied by the command count of the `CommandInteger`. For example, a `CommandInteger` with a `MoveTo` command with a command count of 3 will be followed by 6 `ParameterIntegers`.
+
+A `ParameterInteger` is [zigzag](https://developers.google.com/protocol-buffers/docs/encoding#types) encoded so that small negative and positive values are both encoded as small integers. To encode a parameter value to a `ParameterInteger` the following formula is used:
+
+```
+ParameterInteger = (value << 1) ^ (value >> 31)
+```
+
+Parameter values greater than `pow(2,31) - 1` or less than `-1 * (pow(2,31) - 1)` are not supported.
+
+The following formula is used to decode a `ParameterInteger` to a value:
+
+```
+value = ((ParameterInteger >> 1) ^ (-(ParameterInteger & 1)))
+```
+
+#### 4.3.3. Commands Types
+
+Commands SHOULD NOT be repeated if an increased command count would suffice to produce the same result.
+
+For all descriptions of commands the initial position of the cursor shall be described to be at the point `(cX,cY)` where `cX` is the position of the cursor on the X axis and `cY` is the position of the `cursor` on the Y axis.
+
+##### 4.3.3.1. MoveTo Command
+
+The `MoveTo` command requires two `ParameterIntegers` per command, the first being the `dX` value and the second being the `dY` value. The command moves the cursor to position `pX` and `pY` where:
+
+```
+pX = cX + dX
+```
+
+and
+
+```
+pY = cY + dY
+```
+
+After this command the cursor is at `(pX,pY)`.
+
+The command count for a `MoveTo` command MUST be the number of pairs of `ParameterIntegers` that follow it, and MUST be greater than zero.
```

Comment:
Should something say that command counts greater than 1 are invalid for POLYGON and LINESTRING GeomType?